languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -78796,4 +78796,128 @@ Katie	Katie	EIG:GEN:SIN:FEM:ART:VOR
 Katie	Katie	EIG:NOM:SIN:FEM:ART:VOR
 Katie	Katie	EIG:NOM:SIN:FEM:NOA:VOR
 Katies	Katie	EIG:GEN:SIN:FEM:ART:VOR
-Katies	Katie	EIG:GEN:SIN:FEM:NOA:VOR
\ No newline at end of file
+Katies	Katie	EIG:GEN:SIN:FEM:NOA:VOR
+dabeizuhaben	dabeihaben	VER:EIZ:NON
+herunterzuladen	herunterladen	VER:EIZ:NON
+hinauszugehen	hinausgehen	VER:EIZ:NON
+rauszubringen	rausbringen	VER:EIZ:NON
+rauszulassen	rauslassen	VER:EIZ:NON
+rauszunehmen	rausnehmen	VER:EIZ:NON
+rumzubringen	rumbringen	VER:EIZ:NON
+rumzugeben	rumgeben	VER:EIZ:NON
+rumzugehen	rumgehen	VER:EIZ:NON
+rumzukommen	rumkommen	VER:EIZ:NON
+rumzulaufen	rumlaufen	VER:EIZ:NON
+rumzureiten	rumreiten	VER:EIZ:NON
+rumzurennen	rumrennen	VER:EIZ:NON
+rumzusitzen	rumsitzen	VER:EIZ:NON
+rumzuspinnen	rumspinnen	VER:EIZ:NON
+runterzufahren	runterfahren	VER:EIZ:NON
+runterzuladen	runterladen	VER:EIZ:NON
+runterzuziehen	runterziehen	VER:EIZ:NON
+umherzulaufen	umherlaufen	VER:EIZ:NON
+umherzurennen	umherrennen	VER:EIZ:NON
+voranzutreiben	vorantreiben	VER:EIZ:NON
+wegzurennen	wegrennen	VER:EIZ:NON
+alleinzulassen	alleinlassen	VER:EIZ:NON
+auseinanderzubringen	auseinanderbringen	VER:EIZ:NON
+auseinanderzufallen	auseinanderfallen	VER:EIZ:NON
+auseinanderzugehen	auseinandergehen	VER:EIZ:NON
+auseinanderzuhalten	auseinanderhalten	VER:EIZ:NON
+auseinanderzunehmen	auseinandernehmen	VER:EIZ:NON
+auseinanderzutreiben	auseinandertreiben	VER:EIZ:NON
+fernzuhalten	fernhalten	VER:EIZ:NON
+großzuschreiben	großschreiben	VER:EIZ:NON
+großzuziehen	großziehen	VER:EIZ:NON
+heimzugehen	heimgehen	VER:EIZ:NON
+herauszubringen	herausbringen	VER:EIZ:NON
+herauszufallen	herausfallen	VER:EIZ:NON
+herauszugreifen	herausgreifen	VER:EIZ:NON
+herauszukommen	herauskommen	VER:EIZ:NON
+herauszunehmen	herausnehmen	VER:EIZ:NON
+herauszureden	herausreden	VER:EIZ:SFT
+herauszuschneiden	herausschneiden	VER:EIZ:NON
+hinauszukommen	hinauskommen	VER:EIZ:NON
+hinauszulaufen	hinauslaufen	VER:EIZ:NON
+hinwegzubringen	hinwegbringen	VER:EIZ:NON
+hochzuhalten	hochhalten	VER:EIZ:NON
+hochzuladen	hochladen	VER:EIZ:NON
+hochzuspringen	hochspringen	VER:EIZ:NON
+kleinzuhalten	kleinhalten	VER:EIZ:NON
+rumzuschlagen	rumschlagen	VER:EIZ:NON
+rumzuschmeißen	rumschmeißen	VER:EIZ:NON
+rumzusprechen	rumsprechen	VER:EIZ:NON
+rumzustehen	rumstehen	VER:EIZ:NON
+rumzutreiben	rumtreiben	VER:EIZ:NON
+schwerzunehmen	schwernehmen	VER:EIZ:NON
+überzugehen	übergehen	VER:EIZ:NON
+überzulaufen	überlaufen	VER:EIZ:NON
+zusammenzubekommen	zusammenbekommen	VER:EIZ:NON
+abzuschicken	abschicken	VER:EIZ:SFT
```

Comment:
On a side note, as discussed in our meeting last week, the EIZ-tag is necessary for certain types of missing comma rules. I even question the benefit of applying the 5 tags for "abschicken" to "abzuschicken".

Firstly, it can easily lead to FAs / cases, where you have to work around because of the same tags.

Let R be an XML-rule inserting a comma between two infinitives, then 1) will be correct but 2) will not. There are probably better examples, but I think you get my point.

>1) Ich will diese E-Mail abschicken versucht habe ich schon so viel. (missing comma between "abschicken" and "versucht")
>2) Hier ist die E-Mail, die er abzuschicken versucht. (no missing comma)

Secondly, I don’t see any cases, where applying the same POS-tags to verbs forms like "abzuschicken" is useful for rule developing.

Reply:
@udomai "abzuschicken" is not in the tagger dictionary. It gets the POS tags from schicken (+prefix ab + infix zu). If you add a tag for abzuschicken to added.txt, then the other tags are not generated. `added.txt`does not overwrite by default now in German. Quick solution for this verb: add all needed POS tags to added.txt. 

The repeated tag (VER:EIZ:SFT) is caused by a minor bug in the German tagger (not the main issue here).

Reply:
@affemitkaraffe, the quick solution for this special case is adding all the tags for the whole verb paradigm.

However, this also means that a lot of automatic tagging can be lost for verbs that do not happen to be in a test.

You'd have to check for every EIZ if that eliminates verb tagging like it's the case for "abzuschicken" by checking if that form is in the tagger dict. I've put a dump of it into your Dropbox folder.

All of this might not be worth the effort. How do you assess the immediate cost/benefit ratio here?